### PR TITLE
Regenerate schema.rb file and create missing migrations and fix existing...

### PR DESCRIPTION
**Pull request by zendesk-shender**
Regenerate schema.rb file to allow for less diffs when adding future migrations. Rails 4 changed some of the formatting for it.
I also ran into missing migrations, so I fixed/added those.

/cc @zendesk/runway @samson

### References
 - Jira link:

### Risks
 - None

### CODE REVIEW

File: db/migrate/20150326121012_add_missing_migrations.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class AddMissingMigrations < ActiveRecord::Migration
+  def change
+    create_table :flowdock_flows do |t|
```
Comment (sandlerr) [inline db/migrate/20150326121012_add_missing_migrations.rb]: this should be extracted to a plugin, right?

Comment (zendesk-shender) [inline db/migrate/20150326121012_add_missing_migrations.rb]: Yes, good point, I found the DB migrations in the plugin directories. I'll have to figure out why it didn't pick those up when regenerating the schema.

**Comment by zendesk-shender** [comment]
Added rule to pickup plugin migrations as well. Thanks @sandlerr

File: db/schema.rb [hunk 2]
```diff
@@ -162,41 +162,41 @@
...
     t.datetime "updated_at"
   end
 
-  add_index "stars", ["user_id", "project_id"], name: "index_stars_on_user_id_and_project_id", unique: true, using: :btree
+  add_index "stars", ["user_id", "project_id"], name: "index_stars_on_user_id_and_project_id", unique: true
 
-  create_table "users", force: true do |t|
-    t.string   "name"
+  create_table "users", force: :cascade do |t|
+    t.string   "name",                           null: false
```
Comment (jonmoter) [inline db/schema.rb]: Did you mean to add a non-null constraint to this column? The Travis tests are failing as a result of this.

Comment (zendesk-shender) [inline db/schema.rb]: Yea, looks like that was added expicitly in the EnforceLockUserId migration. I'll update the unit tests to reflect this as it looks like this should be a required field anyway.

Comment (zendesk-shender) [inline db/schema.rb]: Another good reason for this PR! :)
Getting all the migrations in sync and stop manually editing the schema.rb.

**Comment by zendesk-shender** [comment]
ping @zendesk/runway @zendesk/samson

**Comment by jonmoter** [comment]
:+1:

File: db/migrate/20150224031239_add_email_settings_to_stages.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 class AddEmailSettingsToStages < ActiveRecord::Migration
   def change
     add_column :stages, :email_committers_on_automated_deploy_failure, :boolean, default: false, null: false
-    add_column :stages, :static_emails_on_automated_deploy_failure, :string
+    add_column :stages, :static_emails_on_automated_deploy_failure, :string, limit: 255
```
Comment (sandlerr) [inline db/migrate/20150224031239_add_email_settings_to_stages.rb]: I guess its conceivable for this not to be enough? Although maybe by that point its better to setup a mailing list.

**Comment by sandlerr** [comment]
:+1:

**Pull request merged**